uint/tests/uint_tests.rs

```diff
@@ -1151,6 +1151,47 @@ fn trailing_zeros() {
 	assert_eq!(U256::from("0000000000000000000000000000000000000000000000000000000000000000").trailing_zeros(), 256);
 }
 
+#[test]
+fn bit_assign() {
+	fn check(a: U256, b: U256) {
+		// and
+		{
+			let mut x = a;
+			x &= b;
+			assert_eq!(x, a & b);
+		}
+		// or
+		{
+			let mut x = a;
+			x |= b;
+			assert_eq!(x, a | b);
+		}
+		// xor
+		{
+			let mut x = a;
+			x ^= b;
+			assert_eq!(x, a ^ b);
+		}
+		// shr
```

Comment:
It looks like `shr` and `shl` were already implemented and you just added tests for them?

Reply:
That is correct, I couldn't find the tests for them and figured it wouldn't hurt to add them.